python-client/api-reference/usersession.mdx

```diff
@@ -0,0 +1,106 @@
+---
+title: UserSession
+---
+
+Checkout the documentation about [UserSessions](/concepts/usersession) for more information
+
+## Creating a UserSession
+
+```python
+session = await client.api.create_user_session(
+    participant_identifier="<PARTICIPANT_UUID>",
```

Comment:
this is not an uuid this is any id from the monitored app system

Reply:
I don't understand what you mean. It is not a foreign key ?